src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -292,4 +294,8 @@
       <DependentUpon>Splash.resx</DependentUpon>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Does this get referenced even on xamarinmac ?

Reply:
yes